### ISSUE TITLE
Config backup of version 1.4.0 produces strange file name

### ISSUE THREAD

**Issue by Horst12** [marvinweber/KPSimpleBackup#35]
The file is named
KeePass.config.xml.backup-application-config_2021.04.13_20.55.26
The .26 belongs to the date time added but the extension should be .xml

**Comment by marvinweber** [comment]
Thats right, but this shouldn't be a problem. You can simply remove `.backup-application-config_2021.04.13_20.55.26` if you want to restore the file.

Or do you have any other problem with the "wrong" extension?

**Comment by marvinweber** [comment]
However, I'll change to backup with the file extension added at the end, as this allows to open it directly with an xml viewer, etc.

**Comment by Horst12** [comment]
I don't have a big problem with the extension but as you said it opens with no application by default
and it isn't found by some indexing rules I have.
So its better to have the correct extension assigned by default.
Thanks for your very usefull tool.
